core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/visualization/barChart/BarChartOpDesc.scala

```diff
@@ -50,6 +50,17 @@ class BarChartOpDesc extends VisualizationOperator with PythonOperatorDescriptor
   @AutofillAttributeName
   var fields: String = ""
 
+  @JsonProperty(defaultValue = "false", required = false)
```

Comment:
It is done now. Initially, I also wanted to keep only one field but couldn't see the blank option in the drop down list. However, when I closely looked at the drop down list options, I do see a small blank option available at the end of the list due to default value settings.

Reply:
because the blank/empty option in the dropdownlist has a very small space for the user to select therefore I added an option "No Selection" which is easy for the user to select. Hope this will be ok.